pages/common/jq.md

```diff
@@ -0,0 +1,23 @@
+# jq
+
+> A lightweight and flexible command-line JSON processor.
+
+- Output JSON file:
+
+`jq '' {{file}}`
+
+- Read JSON stream into array and output:
```

Comment:
On a slightly tangential note, would it make a difference if the `''` came before the `-s`? If not, I think that'd be more intuitive.